### ISSUE TITLE
Publication extension to Open VSX Registry

### ISSUE THREAD

**Issue by mgerasimchuk** [buenon/scratchpads#44]
Hi @buenon,

I would be happy to use your great extension in the popular VSCode IDE fork named [⭐ 69K code-server](https://github.com/coder/code-server)

By default the **code-server** only returns results from [Open VSX](https://open-vsx.org). Microsoft Terms of Service does not allow programs other than those made by Microsoft to use the VSCode Marketplace. [microsoft/vscode#31168 (comment)](https://github.com/microsoft/vscode/issues/31168#issuecomment-1136375670)

According to the [answer from code-server's maintainers](https://github.com/coder/code-server/discussions/2345#discussioncomment-9135339), you need to publish your extension on https://open-vsx.org, by following this instruction - https://github.com/eclipse/openvsx/wiki/Publishing-Extensions and after this a lot of users of the code-server VSCode IDE will be able to use your extension

Thank you for your solution, and thank you in advance if you will publish it on OVSX

**Comment by cpascual** [comment]
Hi, as a user of [VSCodium](https://vscodium.com/) I was about to request the same.

Of the many extensions I use, scratchpads is the only not in OpenVSX and still I go through the trouble of installing it manually because I find it super-useful 😄 . 

Note that it is possible to request OpenVSX to [automatically mirror a package from the Microsoft Marketplace](https://github.com/EclipseFdn/open-vsx.org/wiki/Auto-Publishing-Extensions.md) (this does not require any changes on this repo or any action on your side).

@buenon, would consider requesting the auto-mirroring? 

...or, alternatively, would you have any objection to me requesting the auto-mirroring myself?

**Comment by buenon** [comment]
Hi guys,
Thanks for bringing it to my attention.
I'll look into it and update shortly.

**Comment by buenon** [comment]
@mgerasimchuk @cpascual I published the extension on OpenVSX.
https://open-vsx.org/extension/buenon/scratchpads

In the process of claiming my namespace.
Let me know if there are any issues.

Thanks again!

**Comment by mgerasimchuk** [comment]
@buenon thank you so much, I tested it right now on code-server, and it all works perfectly!

**Comment by cpascual** [comment]
Yes, thanks @buenon for your quick response! I confirm it is readily available from VSCodium now. Thanks a lot!
